### PR TITLE
Add 3 custom sensor examples

### PR DESCRIPTION
**Pull request by iche033**
Added 3 example custom sensors:
* point lidar
* naive radar
* naive spinning radar

Please also take a look at the accompanying wiki page documentation:
https://github.com/osrf/mbzirc/wiki/Custom-Sensors

The documentation illustrates how new custom sensors can be added to the simulator using 3 approaches :
1. tweaking existing sensor param (point lidar),
2. implementing sensor from ground up (naive radar)
3. adapting existing sensor (naive spinning radar)

These example sensors should also be functional - just need to uncomment out the `install` cmds in the CMakeLists.txt in their respective folders as instructed in the wiki page.

Another change I made is that I bridged the laser scan topics for lidars (we were only bridging point cloud topics).

### CODE REVIEW

File: mbzirc_custom/mbzirc_naive_radar/src/NaiveRadar.cc [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+NaiveRadar::~NaiveRadar() = default;
+
+//////////////////////////////////////////////////
+void NaiveRadar::Configure(const ignition::gazebo::Entity &_entity,
+    const std::shared_ptr<const sdf::Element> &_sdf,
+    ignition::gazebo::EntityComponentManager &_ecm,
+    ignition::gazebo::EventManager &/*_eventMgr*/)
+{
+  // parse configuration parameters from SDF
+  auto sdf = const_cast<sdf::Element *>(_sdf.get());
```
Comment (mjcarroll) [inline mbzirc_custom/mbzirc_naive_radar/src/NaiveRadar.cc:48]: Why was this `const_cast` necessary?

**Comment by mjcarroll** [comment]
The approach looks good and extensible for teams.  I worked through the wiki instructions as well with no issues.

**Review by mjcarroll — APPROVED** [review]

File: mbzirc_custom/mbzirc_naive_radar/src/NaiveRadar.cc (hunk 1 above)
Comment (iche033) [reply mbzirc_custom/mbzirc_naive_radar/src/NaiveRadar.cc]: nope removed. [90641cf](https://github.com/osrf/mbzirc/pull/105/commits/90641cfbd946be5ac560a5358e8ddc659b503584).

Comment (iche033) [reply mbzirc_custom/mbzirc_naive_radar/src/NaiveRadar.cc]: urgh spoke and merged too soon. The cast is needed. Fixed in [49794e4](https://github.com/osrf/mbzirc/pull/110/commits/49794e4c0be7050d487a0fc8dcb6cf3a1ac5e68f) in #110

**Pull request merged**